features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2

```diff
@@ -49,6 +49,9 @@
                 and would be deleted during cleanup task -->
                 <CleanUpTimeout>{{session_data.cleanup.expire_pre_session_data_after}}</CleanUpTimeout>
             </TempDataCleanup>
+            <UserSessionMapping>
+                <Enable>{{session_data.persistence.enable_user_session_mapping}}</Enable>
```

Comment:
This need to be added to the identity.xml as well.

Reply:
The configuration has been already added to the identity.xml file.
Reference: https://github.com/wso2/carbon-identity-framework/blob/master/features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml#L55